src/api.js

```diff
@@ -159,7 +159,37 @@ api.declare({
   });
 });
 
-/** List namespaces inside another namespace */
+/** GET List namespaces inside another namespace */
+api.declare({
+  method:         'get',
+  route:          '/namespaces/:namespace?',
+  name:           'listNamespacesGet',
+  stability:      'deprecated',
+  output:         'list-namespaces-response.json#',
+  title:          'List Namespaces',
+  description: [
```

Comment:
You'll also need to define the query parameters, like
https://github.com/taskcluster/taskcluster-queue/blob/2c8f8f5f07655356cf2d1dcad31730a19270402f/src/api.js#L222-L225

Reply:
Declared query parameters
https://github.com/taskcluster/taskcluster-index/pull/39/files#diff-e1d62d8c4965edf0e532dfa19da0891fR166